packages/contracts/test/contracts/L1/messaging/deposit.gas.spec.ts

```diff
@@ -0,0 +1,182 @@
+/* External Imports */
+import { ethers } from 'hardhat'
+import { Signer, ContractFactory, Contract, constants } from 'ethers'
+import { smoddit } from '@eth-optimism/smock'
+import {
+  AppendSequencerBatchParams,
+  BatchContext,
+  encodeAppendSequencerBatch,
+} from '@eth-optimism/core-utils'
+import { TransactionResponse } from '@ethersproject/abstract-provider'
+import { keccak256 } from 'ethers/lib/utils'
+import _ from 'lodash'
```

Comment:
Do we need `lodash` as a dependency? I view it as high risk when thinking of supply chain attacks

Reply:
It wasn't used in this file. 
It is used in another test, and is not straightforward to remove. I'll create a ticket to track it.